### DISCUSSION TITLE
dwg with block not read by autocad

### DISCUSSION THREAD

**Discussion by rufus1265** [DomCR/ACadSharp#477] [Q&A]
Hi 
i need to create fromn the sctratch a dwg with some blocks into it. To begin i wrote this very simple program 
InsertExamples.CreateInsert();
CadDocument doc = InsertExamples.AddInsertIntoModel();
WriterExamples.WriteDwg(@"c:\temp\test.dwg", doc);
i have this warning printed in the console: Object type not implemented ACadSharp.Objects.MultiLeaderStyle
the file dwg created is not read by the tools we are using like DwgCreator or Autodesk DwgTrueView
attached you can find the dwg file created
Thank you for yiour help
[test.zip](https://github.com/user-attachments/files/17479644/test.zip)

regards

**Comment by DomCR** [comment]
Hi @rufus1265,

DwgCreator or Autodesk DwgTrueView provide a log information about what's failing? 

In Autocad everything seems fine.

**Comment by rufus1265** [comment]
Good morning
Finally, I managed to test on a “real” autocad and it works. Thank you
I need to generate a pdf out of the dwg file. I saw there the ACadSharp.Pdf solution. What is its development status? Could it be used?
Thank you
Regards
Neri alamanni

**Reply by DomCR** [reply]
I'm planning to release a Beta for that project shortly once is stable, there will be some missing features like some entities like Hatch or Text that will be implemented in the future or page configurations.
